src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -101,7 +106,18 @@ public CheckInProcessRecords withHighestPriorityFulfillableRequest(Request reque
       this.loan,
       this.requestQueue,
       this.checkInServicePoint,
-      request);
+      request, loggedInUserId);
+  }
+
+  public CheckInProcessRecords withLoggedInUserId(String staffMemberId) {
```

Comment:
Why is this parameter named differently to the method name, is it the logged in user or the staff member?

Reply:
done.